### DISCUSSION TITLE
Use other environment variables instead of ES_HOST and such

### DISCUSSION THREAD

**Discussion by SebPikPik** [jertel/elastalert2#1538] [Q&A] [answered]
Hi !

I'd need to use environment variables for both config and rules. Some rules would need to query $HOST1 and some other $HOST2

when I try in either config.yaml or rules files to input es_host: $HOST1 or es_host: $HOST2, it doesn't work.

Is it possible?

**Comment by jertel** [comment, marked as answer]
ElastAlert 2 is designed to use a single Elastic host. It not only queries event data from Elastic but also writesback status data. Using multiple Elastic instances with a single ElastAlert 2 instance isn't supported.
